cern_access/indico_cern_access/templates/identity_data_form_email.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'events/registration/emails/base_registration_details.html' %}
```

Comment:
do we actually want to include the whole registration summary in this email? if not, better inherit from `events/registration/emails/base_registration.html`